### PR TITLE
TWON-18165: Increase battery theshold to shutdown old devices

### PR DESCRIPTION
**Pull request by tpaschidis**

### CODE REVIEW

File: drivers/power/bd7181x-power.c [hunk 1]
```diff
@@ -135,7 +138,7 @@ static const struct tn_power_values_st TN_POWER_CROSS = {
 	.term_current = 0x06,
 	.fast_charge_current = 0x0A, // 1A -> 0x0A (100mA steps)
 	.capacity = 3300,
-	.low_voltage_th = 0x00D6, // 0x00D6 (214) * 16mV = 3.4297V,
+	.low_voltage_th = 0x00D4, // 3400 / 16mV = 212 -> 0x00D4
```
Comment (lluisdiaz) [inline drivers/power/bd7181x-power.c]: has bajado el th? el objetivo no era subirlos?

File: drivers/power/bd7181x-power.c [hunk 2]
```diff
@@ -294,26 +296,44 @@ static u32 rsense_current_factor = 1000; // (/ factor)
...
+	// Kingston 16GB: TB2916
+	// Kingston 32GB: TA2932
+	// Toshiba 16GB:  016G30
+	*/
+
 	if(strstr(hwtype, "trailplus") != NULL) {
 		tn_power_values = TN_POWER_TRAIL;
-		rsense_capacity_factor = 360; // TODO: not known yet
-		rsense_current_factor = 1000; // TODO: not known yet
+		rsense_capacity_factor = 248; // verify
```
Comment (lluisdiaz) [inline drivers/power/bd7181x-power.c:309]: why this values are not contained inside the same device struct as others? (just commenting as a possible upgrade)

File: drivers/power/bd7181x-power.c (hunk 1 above)
Comment (tpaschidis) [reply drivers/power/bd7181x-power.c]: En el Cross el threshold ya estaba ajustado. Podría estar almacenazo 3.7 meses y con este cambio puede estar 3.5meses y se gana un poco de autonomia, tiene la bateria mas pequeña. Los problematicos fueron Trail y Aventura segun el codigo.

Comment (tpaschidis) [reply drivers/power/bd7181x-power.c]: Ya lo he puesto como estaba

File: drivers/power/bd7181x-power.c (hunk 2 above)
Comment (tpaschidis) [reply drivers/power/bd7181x-power.c:309]: For the time being the battery is the same for Trail and TrailPlus but the Rsense correction is different thus to not duplicate the structure i let it seperatelly. We can loot at it on another iteration

Comment (lluisdiaz) [reply drivers/power/bd7181x-power.c:309]: On the next change it'll be convenient to group those values in the same struct, as they could be different on other devices ( cross for example )

**Review by lluisdiaz — APPROVED** [review]

**Pull request merged**